compiler-plugin/src/main/kotlin/arrow/meta/internal/registry/InternalRegistry.kt

```diff
@@ -154,7 +152,7 @@ interface InternalRegistry : ConfigSyntax {
     project: Project,
     configuration: CompilerConfiguration
   ) {
-    println("Project allowed extensions: ${(Extensions.getArea(project) as ExtensionsAreaImpl).extensionPoints.toList().joinToString("\n")}")
+    //println("Project allowed extensions: ${(project.extensionArea as ExtensionsAreaImpl).extensionPoints.toList().joinToString("\n")}")
```

Comment:
this should be re-enabled methinks

Reply:
Thanks for the review @pakoito :raised_hands: API changes in that version made no possible to get those data so we decided to comment it meanwhile...